website/docs/cli/commands/describe/describe-affected.md

```diff
@@ -0,0 +1,112 @@
+---
+title: atmos describe affected
+sidebar_label: affected
+sidebar_class_name: command
+id: affected
+description: This command produces a list of the affected Atmos components and stacks given two Git commits.
+---
+
+:::note Purpose
+Use this command to show a list of the affected Atmos components and stacks given two Git commits.
+:::
+
+<br/>
+
+:::info
+For the first commit, the command assumes that the repo root is a Git checkout.
+
+The second commit is specified on the command line by using the `--ref` or `--sha` flags.
+
+If the flags are not provided, it will clone the HEAD of the default branch.
```

Comment:
```suggestion
If the flags are not provided, the `ref` be the default branch (e.g. `main`) and the `sha` will point to the `HEAD` of the branch.
```

Reply:
The fact that we clone it, will always be the case. I think we should describe our logic of how this works in plain English.

This command works by:
1. Cloning the branch (`--ref`) and checking out the commit (`--sha`) of the target branch. 
2. Deep merge all configurations for both the current working branch and the target branch
3. Compare each section of the stack configuration looking for differences
4. Look for changes in the component directories
5. De-dupe affected stacks
6. Output a JSON document consisting of a list of affected stacks and what caused it to be affected